src/asciidoctor-core-api.js

```diff
@@ -94,6 +94,14 @@ Opal.Asciidoctor.AbstractBlock.$$proto.convert = function () {
   return this.$convert();
 };
 
+Opal.Asciidoctor.AbstractBlock.$$proto.getLineno = Opal.Asciidoctor.AbstractBlock.$$proto.getLineNumber = function () {
```

Comment:
I think we should just drop `getLineno`. We only keep `getDoctitle` since "doctitle" is so core to the terminology of AsciiDoc. But there's no such understanding for "lineno" so I think we should just use the formal term, `getLineNumber`.